src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,75 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.File;
+import java.io.FileReader;
+import java.io.IOException;
+import java.nio.file.Files;
+
 public class WorkWithFile {
+    private static final int INDEX_DATA = 0;
+    private static final int INDEX_MONEY = 1;
+    private static final String COMMA = ",";
+    private static final String SEPARATOR = System.lineSeparator();
+    private static final String SUPPLY = "supply";
+    private static final String BUY = "buy";
+
     public void getStatistic(String fromFileName, String toFileName) {
+        String data = readFromFile(fromFileName);
+        String report = createReport(data);
+        writeToFile(report, toFileName);
 
     }
+
+    public String readFromFile(String fromFileName) {
+        File file = new File(fromFileName);
+        try {
+            BufferedReader bufferedReader = new BufferedReader(new FileReader(file));
```

Comment:
use try with resources
```suggestion
        try {
            BufferedReader bufferedReader = new BufferedReader(new FileReader(fromFileName));
```